src/libsyntax/feature_gate/builtin_attrs.rs

```diff
@@ -167,7 +167,7 @@ macro_rules! experimental {
 }
 
 const IMPL_DETAIL: &str = "internal implementation detail";
-const INTERAL_UNSTABLE: &str = "this is an internal attribute that will never be stable";
+const INTERNAL_UNSTBLE: &str = "this is an internal attribute that will never be stable";
```

Comment:
Why did this name change?

Reply:
A typo fix

Reply:
It looks like it changed from one typo to another. Shouldn't it be `INTERNAL_UNSTABLE`?

Reply:
Argh.. Didn't notice. Will fix. Can you review the rest of the PR meanwhile?